public/js/singup.js

```diff
@@ -0,0 +1,13 @@
+const backBtn = document.querySelector('.back');
```

Comment:
There is a typo in this filename (signup.js). Also it doesn't seem that this script is called anywhere so it may not be doing anything. I believe this script is not needed though.

The back button provides a hyperlink to the index page. And the submit button redirects to `/flix` once the user is saved (see `controllers/auth.js` line 101: `res.redirect('/flix')`)